zcash_client_sqlite/src/wallet/common.rs

```diff
@@ -169,7 +169,7 @@ pub(crate) fn get_nullifiers<N, F: Fn(&[u8]) -> Result<N, SqliteClientError>>(
             "SELECT a.uuid, rn.nf
                  FROM {table_prefix}_received_notes rn
                  JOIN accounts a ON a.id = rn.account_id
-                 JOIN transactions tx ON tx.id_tx = rn.tx
+                 JOIN transactions tx ON tx.id_tx = rn.transaction_id
```

Comment:
Have you confirmed that none of the modified queries are used in any prior migration? If any are, we need to first archive identical versions of the queries within those migrations, before modifying the queries for the renamed columns.

CI / testing is unlikely to spot this, because it will be the data migrations that break.